### PR TITLE
Doc/add a description of Flatten

### PR DESCRIPTION
**Pull request by L-M-Sherlock**
I will update the add-on page after merging it.

### CODE REVIEW

**Comment by user1823** [comment]
I am really not convinced how this feature is useful and until I am convinced, I can't give proper feedback.

I explained my points here: https://github.com/open-spaced-repetition/fsrs4anki-helper/issues/439#issuecomment-2268740000

**Comment by Expertium** [comment]
It's useful for keeping your workload as consistent as possible. That being said, we already have the Load Balancer, and Flatten will mess up retention way too much. So I'm more on user1823's side: maybe we don't need Flatten at all.

**Comment by L-M-Sherlock** [comment]
Fine. This feature is required by #439. If you don't have suggestions to the description, I will merge this feature tomorrow.

By the way, this feature is popular in my company's app because users like placebo and they even pay for it.

File: README.md [hunk 1]
```diff
@@ -2,23 +2,24 @@
...
 - **Balance** the load during rescheduling (based on fuzz).
-- **Less Anki** on Easy Days (such as weekends) during rescheduling (based on load balance).
+- Less Anki on **Easy Days** (such as weekends) during rescheduling (based on load balance).
 - **Disperse** Siblings (cards with the same note) to avoid interference & reminder.
+- **Flatten** future due cards to a selected number of reviews per day.
 
 # Requirements
 
-- For Anki version in 2.1.55 - 2.1.66 
+- For Anki version in 2.1.55 - 2.1.66 (maintenance is stopped)
```
Comment (user1823) [inline README.md:17]:
```suggestion
- For Anki version in 2.1.55 - 2.1.66 (no longer maintained)
```

File: README.md [hunk 2]
```diff
@@ -2,23 +2,24 @@
...
   - Enable V3 Scheduler
-  - FSRS4Anki scheduler version >= 3.0.0
+  - FSRS4Anki version >= 3.0.0
 - For Anki version >= 23.10
```
Comment (user1823) [inline README.md:20]: Move this above the 2.1.55-2.1.66 part. I believe that the latest version should always be at the top, especially if the older version is no longer maintained.

File: README.md [hunk 3]
```diff
@@ -2,23 +2,24 @@
...
-  - Enable built-in FSRS
-  - Remove FSRS4Anki scheduler code if you used it before
+  - Enable built-in FSRS (follow this [tutorial](https://github.com/open-spaced-repetition/fsrs4anki/blob/main/docs/tutorial.md))
+  - Remove FSRS4Anki custom scheduling code if you used it before
```
Comment (user1823) [inline README.md:22]:
```suggestion
  - Remove FSRS4Anki custom scheduling code if you have used it before
```

File: README.md [hunk 4]
```diff
@@ -94,6 +96,10 @@ In Anki, some templates will generate multiple cards related in content from the
 
 ![image](https://github.com/open-spaced-repetition/fsrs4anki-helper/assets/32575846/b33460f4-d7db-4c8f-b9d0-d0193f2d1f54)
 
+## Flatten
+
+You can the maximum number of reviews per day for future due cards. The Helper will find out the cards that exceed the limit and postpone them to the future.
```
Comment (user1823) [inline README.md:101]:
```suggestion
You can set the maximum number of reviews per day for future days. The Helper add-on will find out the days that exceed the limit and postpone the extra cards to the future.
```

**Pull request merged**